packages/kit/src/core/adapt/prerender.js

```diff
@@ -330,11 +329,10 @@ export async function prerender({ cwd, out, log, config, build_data, fallback, a
 	if (fallback) {
 		const rendered = await app.render(
 			{
+				url: `${config.kit.host || 'sveltekit'}://${config.kit.host || 'prerender'}/[fallback]`,
```

Comment:
Is the `[fallback]` thing still necessary here? Is anything looking at that?

Reply:
the string is mostly there to keep the typechecker happy. i chose this in case someone logged it out in `__layout` for some reason — probably totally unnecessary but couldn't think of anything more suitable